src/routes/playground/tags/playground/tags/result/result.style.module.scss

```
@@ -193,6 +193,42 @@
 
+.loading {
+  position: absolute;
+  inset: 0;
+  z-index: 10;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: var(--color-background);
+  font-family: "Ubuntu Mono", monospace;
+  font-size: 1.25rem;
+  // Stay invisible briefly so fast loads never flash the indicator.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)